### ISSUE TITLE
3rd party imports use the "src" directory

### ISSUE THREAD

**Issue by ResoDev** [Milad-Akarie/auto_route_library#34]
Hello Milad!
The same problem that happened in **injectable** mentioned in [the issue no. 12](https://github.com/Milad-Akarie/injectable/issues/12) also happens in **auto_route** with route arguments.

```dart
  final Option<Note> editedNoteOption;

  const NoteFormPage({
    Key key,
    @required this.editedNoteOption,
  }) : super(key: key);
```
generates the following erroneous import
```dart
import 'package:dartz/src/option.dart';
```

**Comment by Milad-Akarie** [comment]
Hey, @ResoDev ya I'll just apply the same solution from injectable.

**Comment by Milad-Akarie** [comment]
Hey Reso
Fixed in v0.3.1.
Thanks for reporting.
